### PR TITLE
feat: [EXAM-2555] add api to check that all exams are completed

### PR DESCRIPTION
**Pull request by YuriiBorovenskyi**
**Description:**
* add api to check that all exams are completed

**[YouTrack:** https://youtrack.raccoongang.com/issue/EXAM-2555](https://youtrack.raccoongang.com/issue/EXAM-2555)

**Merge checklist:**
* [x] Demo status: OK

**Post-Merge:**
* [ ] Create `ukr-sertification-juniper-v.2.10.3` tag matching the new version number. (Will be made after merge)

### CODE REVIEW

File: edx_proctoring/api.py [hunk 1]
```diff
@@ -1339,6 +1339,17 @@ def get_last_exam_completion_date(course_id, username):
     return exam_attempts[0].completed_at if exam_attempts and are_all_exams_attempted else None
 
 
+def are_all_exams_completed(user_id, course_id):
+    """
+    Returns True if all needed student exam attempts for given course are completed else False.
+    """
+    all_exams = ProctoredExam.get_all_exams_for_course(course_id, active_only=True)
+    completed_exams_count = ProctoredExamStudentAttempt.objects.get_completed_student_attempts(
```
Comment (NikolayBorovenskiy) [inline edx_proctoring/api.py]: `completed_exams`

File: edx_proctoring/api.py [hunk 2]
```diff
@@ -1339,6 +1339,17 @@ def get_last_exam_completion_date(course_id, username):
...
+        user_id=user_id, course_id=course_id,
+    ).count()
+    return completed_exams_count == all_exams.count()
```
Comment (NikolayBorovenskiy) [inline edx_proctoring/api.py]: `completed_exams.count() == all_exams.count()`

**Review by NikolayBorovenskiy — CHANGES_REQUESTED** [review]

File: edx_proctoring/api.py (hunk 1 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/api.py]: Fixed.

File: edx_proctoring/api.py (hunk 2 above)
Comment (YuriiBorovenskyi) [reply edx_proctoring/api.py]: Fixed.

**Review by NikolayBorovenskiy — APPROVED** [review]

**Pull request merged**